source/guide_ts3audiobot.rst

```diff
@@ -0,0 +1,264 @@
+.. highlight:: console
+
+.. author:: Felix Franz <https://www.felix-franz.com>
+
+.. tag:: web
+
+.. _TS3AudioBot: https://github.com/Splamy/TS3AudioBot
+.. _TeamSpeak3: https://teamspeak.com
+.. _youtube-dl: https://github.com/ytdl-org/youtube-dl
+
+###########
+TS3AudioBot
+###########
+
+.. tag_list::
+
+TS3AudioBot_ is a open-source TeamSpeak3 bot, playing music and much more.
+
+.. warning:: For using the bot, you also need a TeamSpeak3_ server!
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * TeamSpeak3_
+  * :manual:`.htaccess <web-documentroot>`
+  * :manual:`supervisord <daemons-supervisord>`
+..  * :manual:`domains <web-domains>`
+  * :manual:`lang-dotnet`
+
+
+License
+=======
+
+TS3AudioBot_ was written by `Splamy <https://splamy.de/>`_ and licensed under `OSL-3.0 <https://github.com/Splamy/TS3AudioBot#license>`_.
+
+
+Installation
+============
+
+Download release file:
+
+::
+
+ [isabell@stardust ~]$ mkdir ts3audiobot
+ [isabell@stardust ~]$ cd ts3audiobot
+ [isabell@stardust ts3audiobot]$ wget https://github.com/Splamy/TS3AudioBot/releases/latest/download/TS3AudioBot_dotnet_core_3.1.zip
+
+Extract zip archive:
+
+::
+
+ [isabell@stardust ts3audiobot]$ unzip TS3AudioBot_dotnet_core_3.1.zip
+ [isabell@stardust ts3audiobot]$ rm TS3AudioBot_dotnet_core_3.1.zip
+
+Now we will configure the bot.
+To set your TeamSpeak user as bot admin, you unique id is required.
+Go to your TeamSpeak client and open the `Identities` window in the `Tools` menu.
+Copy your `unique id`.
+
+To configure the bot, start it an interactively insert your configuration data (unique TeamSpeak user id, server name, server password, ...).
+After the configuration is done the bot will automatically connect to the TeamSpeak3 server.
+
+::
+
+ [isabell@stardust ts3audiobot]$ dotnet TS3AudioBot.dll
+
+.. warning::
+
+ The bot also starts a webinterface by default on port 58913. If the port is already used you need to change this port.
+
+ If you see such a error message stop the bot by ``Ctrl+C`` and configure another port.
+ You might re-try this process until you reach a freee port.
+ Change ``~/ts3audiobot/ts3audiobot.toml`` from
+
+ .. code-block:: toml
+
+  port = 58913
+
+ to
+
+ .. code-block:: toml
+
+  port = <another port>
```

Comment:
Each guide should assume that the target uberspace is empty, so the port will be free and we don't need to handle this case here.
```suggestion
```